### DISCUSSION TITLE
Add new source from Hungary/Erd

### DISCUSSION THREAD

**Discussion by sthevenp** [mampfes/hacs_waste_collection_schedule#1339] [Q&A]
Dear All,

Is there any chance to add this waste collection schedule service:

https://www.eth-erd.hu/hulladeknaptar

This page looks similar to this one: 
FKF Budapest, HU (tahnks @TMShader)

Thanks in advance.

**Comment by tmshader** [comment]
Hi!
I looked into it and I think I can implement it when I have a bit of time.

**Comment by 5ila5** [comment]
opened issue #1341 as nearly all source requests are issues.

**Reply by 5ila5** [reply]
added in #1344
